internal/server/v2/propertyvalues/golden/simple/california.json

```diff
@@ -0,0 +1,48 @@
+{
+  "data": {
+    "geoId/06": {
+      "nodes": {
+        "administrativeCapital": {
```

Comment:
note here: we would say "geoId/06" has some "arcs" with keys "administrativeCapital", "name" etc. The value of each arc is "a list of nodes" @shifucun

Reply:
Note this is not the optimal version where using "nodes" does not convey the idea of "arcs" as in my initial PR version